integr8ly/cloud-resource-operator
# remove ability to update allocated storage via strategy map

add ability to update max allocated storage via strategy map

Currently, aws will autoscale as needed from allocated storage up to max allocated storage value. 
We need to remove the ability to allow cro to update the allocated storage value, as we want to rely 
on aws auto scaling. 
We do wish to allow cro the ability to update max allocated storage value

## Overview

Jira: https://issues.redhat.com/browse/INTLY-7231

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make cluster/seed/managed/postgres
- Run `make run`
- Update `"AllocatedStorage":50` in the create strategy for postgres in the cloud-resources-aws-strategies config map
- Update `"MaxAllocatedStorage":500` in the create strategy for postgres in the cloud-resources-aws-strategies config map
- Ensure allocated storage is not updated (ensure it is not added to pending modifications) 
- Ensure MaxAllocatedStorage is updated to 500 GiB
